DEVELOPING.md

```diff
@@ -704,4 +706,8 @@ WSL 2 version of docker-for-windows.
 Additionally, if your hostname contains an upper-case character, the build script will break. This is based on the
 `NAME` environment variable, which should contain your hostname. You can solve this issue by doing `export NAME=my-lowercase-host-name`.
 If you do this *after* you've already run `make images` once, you will manually have to clean up the docker images
-that have been created using your upper-case hostname.
+that have been created using your upper-case host name.
```

Comment:
Proper resolution of this merge-conflict would have been to take the single-word "hostname" version.